src/ffi.ts

```diff
@@ -16,27 +16,50 @@ export function unload() {
     lib.close();
 }
 
-let lib_file;
+const lib_file = (await getLibFile()).default;
 
-if (process.platform === "win32") {
-    //@ts-expect-error
-    lib_file = await import("../build/libwebview.dll");
-} else if (process.platform === "linux" && process.arch === "x64") {
-    //@ts-expect-error
-    lib_file = await import("../build/libwebview.so");
-} else if (process.platform === "darwin") {
-    if (process.arch === "x64") {
-        //@ts-expect-error
-        lib_file = await import("../build/libwebview.x64.dylib");
-    } else {
-        //@ts-expect-error
-        lib_file = await import("../build/libwebview.arm64.dylib");
-    }
-} else {
-    throw `unsupported platform: ${process.platform}-${process.arch}`;
+/**
+ * It determines the appropriate lib file for the given platform and environment.
+ *
+ * A `string` file location is the technically requirement, however to
+ * trigger bun to bundle the lib files as external resources, `import` is
+ * called here, which also returns the `default` string property for the path.
+ *
+ * If `process.env.WEBVIEW_PATH` is preferred, listing the default lib (absolute)
+ * file paths in `Bun.build({ external: []}` will exclude them. Be sure to
+ * manually include your custom lib file in your distribution bundle.
+ */
+function getLibFile(): Promise<{ default: string }> {
+	if (!!process.env.WEBVIEW_PATH) {
+		return Promise.resolve().then(() => ({
```

Comment:
I am specifically doing this so that the `WEBVIEW_PATH` is relative to the project root, NOT the `node_module/webview-bun/src` folder. The way you have it implemented, the user has to set `WEBVIEW_PATH=../../../user_specified.so`, instead of the much better, and default expectation of `WEBVIEW_PATH=user_specified.so`.

@tr1ckydev Please do not discard code improvements, especially when you do not give reason.